cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/DataLayer.java

```diff
@@ -198,6 +199,17 @@ public SparkRangeFilter sparkRangeFilter(int partitionId)
         return null;
     }
 
+    /**
+     * Returns a list of {@link SSTableTimeRangeFilter} to filter out SSTables based on min and max timestamp.
+     *
+     * @return list of {@link SSTableTimeRangeFilter} for filtering SSTables based on max and min timestamps. When
+     * the list is empty, all SSTables are included for bulk reading.
+     */
+    public List<SSTableTimeRangeFilter> sstableTimeRangeFilters()
```

Comment:
Please update the return type. Just return the filter, instead of a list of filters.